src/app/components/filters/cutstring.filter.js

```diff
@@ -0,0 +1,19 @@
+angular.module('transmartBaseUi').filter('cutstring', function () {
```

Comment:
Okay, now I understand what is this for. I would say it's better to have the side bar is draggable so you can resize the width but it's okay for now. In my opinion user still can't see the full title of a node.